CODEOWNERS

```diff
@@ -139,6 +139,7 @@
 /dts/arm/st/                              @erwango
 /dts/arm/nordic/                          @ioannisg @carlescufi
 /dts/arm/nxp/                             @MaureenHelm
+/dts/bindings/                            @galak
```

Comment:
I can't remember how the searching works with github, should the most generic option be last in the last?  Ie so any CAN bindings would get tagged for @alexanderwachter ?

Reply:
The last matching pattern wins